app/presenters/publishing_api/organisation_presenter.rb

```diff
@@ -81,14 +109,198 @@ def image
       }
     end
 
-    def brand
-      brand_colour = item.organisation_brand_colour
-      brand_colour ? brand_colour.class_name : nil
+    def foi_exempt
+      item.foi_exempt
+    end
+
+    def corporate_information_pages
+      item.published_corporate_information_pages.map do |cip|
+        {
+          title: cip.title,
+          href: Whitehall.url_maker.public_document_path(cip)
+        }
+      end
+    end
+
+    def featured_links
+      item.featured_links.map do |link|
```

Comment:
Do these come in any particular order?

Reply:
I'm using `visible_featured_links` now which is what the controller uses without any extra ordering.